CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.1.1](https://github.com/voxpupuli/puppet-ldapquery/tree/v3.1.1) (2025-03-28)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-ldapquery/compare/v3.1.0...v3.1.1)
+
+**Implemented enhancements:**
+
+- metadata.json: Add OpenVox [\#63](https://github.com/voxpupuli/puppet-ldapquery/pull/63) ([jstraw](https://github.com/jstraw))
```

Comment:
@bastelfreak Should we `skip-changelog` on these?  I'm leaning towards 'yes' as I'd consider it 'noise'. (There are probably better places to shout about what `OpenVox` progress is being made).

Reply:
I thought it's nice to show it in the changelog. And it has quite a big impact to the module because it will run all beaker jobs for openvox as well now. I don't have a strong opinion here, if you prefer to to add `skip-changelog` that's fine to me, but then someone should label https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+user%3Avoxpupuli+%22metadata.json%3A+Add+OpenVox%22+

Reply:
ok.  I'll leave as is.